backend/dal/Migrations/v1.0.14.1/Up/PostUp/02_CLAIMS.sql

```diff
@@ -0,0 +1,72 @@
+declare @sysadmin bigint,@rem bigint,@rea bigint;
```

Comment:
for future reference, this is how to add new claims, and associate those claims with existing roles in the DB. This allows the keycloak sync tool to add those claims and create the associations in keycloak.